### PR TITLE
Dennis nc news 4

### PR DESCRIPTION
**Pull request by DennisTockan**

### CODE REVIEW

File: models/articles.models.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({rows}) => {
+        if (rows.length === 0){
+            return Promise.reject({status: 404, message: 'Not Found'})
```
Comment (johnhutcheon) [inline models/articles.models.js:7]: Nicely done!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -33,3 +33,38 @@ describe("GET /api/topics", () => {
...
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200 accepts an article_id which responds with the specified article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        const {article}  = body;
+          expect(article).toHaveProperty("article_id", expect.any(Number));
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: As we're sending the request to a specific article id, should we be expecting any number here, or something else?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hi Dennis! Your code is really good and you're nearly there! Could you just amend the test I've commented on please? Thanks! 🙂

**Review by robert112233 — APPROVED** [review]
Good to merge, well done!

**Pull request merged**